crates/sui-framework/packages/bridge/sources/bridge.move

```diff
@@ -143,6 +147,34 @@ module bridge::bridge {
         transfer::share_object(bridge);
     }
 
+    #[allow(unused_function)]
+    fun init_bridge_committee(
+        self: &mut Bridge,
+        system_state: &mut SuiSystemState,
+        min_stake_participation_percentage: u64,
+        ctx: &TxContext
+    ) {
+        assert!(tx_context::sender(ctx) == @0x0, ENotSystemAddress);
+        let inner = load_inner_mut(self);
+        if (vec_map::is_empty(committee::committee_members(&inner.committee))) {
```

Comment:
this "should" never happen because in EoE txn we checks the committee object in the object store before adding this call to the txn, however I added this here to make sure we have this safety guard in move code as well.